skbio/embedding/__init__.py

```diff
@@ -0,0 +1,29 @@
+r""" Biological Embeddings (:mod:`skbio.embedding`)
+=================================================
+
+.. currentmodule:: skbio.embedding
+
+This module provides support for storing embeddings for biological objects, such
+as protein embeddings outputted from protein language models (pLMs).
+
+Embedding types
+---------------
+
+.. autosummary::
+    :toctree: generated/
+
+    ProteinEmbedding
```

Comment:
Its a thought, and it does raise questions about the difference between a `ProteinEmbedding` and a `SequenceEmbedding` -- it suggests that there should be protein-specific checks in place. This would be one of the benefits of having the `ProteinEmbedding` object inherent from `Protein`, it would enable validation of the input string (i.e. confirm that they are valid amino acids via IUPAC).

Reply:
For now I think it's okay, other sequence embeddings are not that popular yet. It can always be generalised and abstracted if users show interest to such functionality/